### PR TITLE
expose the editor instance

### PR DESCRIPTION
**Pull request by rotimi-best**
This PR addresses this issue: https://github.com/TGlide/novel-svelte/issues/3

Please let me know if there is anything else I failed to consider for this particular issue.

Thanks.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 	import Nav from './nav.svelte';
-	
 
 	let saveStatus = 'Saved';
+	let editor: EditorType;
+
+	onMount(() => {
+		// Use the tiptap editor instance
+		console.log({ editor });
+	});
```
Comment (TGlide) [inline src/routes/+page.svelte:13]: We don't want the actual docs to have a console.log

File: src/lib/ui/editor/index.svelte [hunk 2]
```diff
@@ -66,9 +66,12 @@
 	 * Defaults to "novel__content".
 	 */
 	export let storageKey = 'novel__content';
+	/**
+	 * The Editor instance created onMount
+	 */
+	export let editor: Editor | undefined = undefined;
```
Comment (TGlide) [inline src/lib/ui/editor/index.svelte:72]:
```suggestion
	/**
	 * The editor instance. Bind to it to get access to the editor.
	 * @example
	 * <script lang="ts">
	 * 	import { type EditorType, Editor } from 'novel-svelte';
	 * 	let editor: EditorType;
	 * </script>
	 * 
	 * <Editor bind:editor />
	 */
	export let editor: Editor | undefined = undefined;
```

**Review by TGlide — COMMENTED** [review]
Thank you for the PR!

File: src/routes/+page.svelte (hunk 1 above)
Comment (rotimi-best) [reply src/routes/+page.svelte]: Done

**Review by TGlide — APPROVED** [review]

**Pull request merged**